asserts/model.go

```diff
@@ -339,6 +339,25 @@ const (
 	ModelDangerous ModelGrade = "dangerous"
 )
 
+// ModelStorageSafety characterizes the requested storage safety of
+// the model which then controls what encryption is used
+type ModelStorageSafety string
+
+const (
+	ModelStorageSafetyUnset ModelStorageSafety = "unset"
```

Comment:
should we have a doc-comment on unset too?

Reply:
we might not need the Model prefix on these, either this will only exist on model or if we have the same field used somewhere else we would do it only if it can have the same values (or at least  a sane subset)